Content.Shared/Holiday/HolidaySystem.cs

```diff
@@ -1,30 +1,29 @@
 using System.Linq;
-using Content.Server.Chat.Managers;
-using Content.Server.GameTicking;
 using Content.Shared.CCVar;
-using Content.Shared.Holiday;
 using Robust.Shared.Configuration;
+using Robust.Shared.Network;
 using Robust.Shared.Prototypes;
 
-namespace Content.Server.Holiday
+namespace Content.Shared.Holiday
 {
     public sealed class HolidaySystem : EntitySystem
```

Comment:
```suggestion
    public abstract partial class SharedHolidaySystem : EntitySystem
```
naming convention
rename the file accordingly
the server and client systems inheriting should be simply named `HolidaySystem`